### PR TITLE
fix(kong) wait for

### PR DESCRIPTION
**Pull request by jackkav**
#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - re-adds wait container, unintentionally removed.
  - fixes issues in the docs

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] New or modified sections of values.yaml are documented in the README.md

### CODE REVIEW

**Comment by wongstein** [comment]
sad that we still need to check for kong womp womp womp

**Review by wongstein — APPROVED** [review]

**Comment by wongstein** [comment]
@lena-larionova can you also check the README.md changes and make sure they are sane?

File: README.md [hunk 1]
```diff
@@ -28,10 +28,7 @@ deployment on a [Kubernetes](http://kubernetes.io) cluster using the
 
 To install the chart with the release name `my-release`:
 
-1. When Kong, make sure to set the IMMUNITY_ENABLED to true.  This does have to be done on Kong installation, and it is necessary to see the Immunity page on Kong Manager.
-```console
-helm install kong/kong --version=$KONG_HELM_VERSION --set IMMUNITY_ENABLED=true
-```
+1. When Kong, make sure that admin_gui_flags includes an "IMMUNITY_ENABLED" set to true.  An example of this can be found in kong-values.yaml file in this repo.  This does have to be done on Kong installation, and it is necessary to see the Alerts page on Kong Manager.
```
Comment (lena-larionova) [inline README.md:31]:
```suggestion
1. When installing Kong, make sure that `admin_gui_flags` includes the `IMMUNITY_ENABLED` flag set to `true`. This must be done on Kong installation, and it is necessary to see the Alerts page on Kong Manager.
    
    You can find an example of this in the [kong-values.yaml](https://github.com/Kong/kong-collector-helm/blob/master/kong-values.yaml) file in this repo. 
```

File: README.md [hunk 2]
```diff
@@ -40,8 +37,7 @@ $ kubectl create secret generic kong-admin-token-secret --from-literal=kong-admi
 secret/kong-admin-token-secret created
 ```
 
-3. Set up collector, overriding Kong Admin host, servicePort and token to ensure
-   Kong Admin API is reachable by collector.
+3. Set up collector, overriding Kong Admin host, servicePort and token to ensure Kong Admin API is reachable by collector.
```
Comment (lena-larionova) [inline README.md:40]:
```suggestion
3. Set up collector, overriding Kong Admin host, servicePort, and token to ensure Kong Admin API is reachable by collector.
```

**Pull request merged**